vllm/_custom_ops.py

```diff
@@ -1215,8 +1215,12 @@ def scaled_fp8_quant(
             torch.ops._C.dynamic_per_token_scaled_fp8_quant(
                 output, input, scale, scale_ub)
         else:
-            scale = torch.empty(1, device=input.device, dtype=torch.float32)
-            torch.ops._C.dynamic_scaled_fp8_quant(output, input, scale)
+            scale = torch.zeros(1, device=input.device, dtype=torch.float32)
```

Comment:
I remember one previous PR specifically change to empty, will this cause trouble?

Reply:
Yes. XPU quantization kernel has problem with empty scale. I separate this value to keep empty for non-xpu path, okay for you?

Reply:
Updated.